packages/core/saved-objects/core-saved-objects-migration-server-internal/src/kibana_migrator.ts

```diff
@@ -39,11 +39,12 @@ import { DocumentMigrator } from './document_migrator';
 import { runZeroDowntimeMigration } from './zdt';
 import { ALLOWED_CONVERT_VERSION } from './kibana_migrator_constants';
 import { runV2Migration } from './run_v2_migration';
-
+import * as HASH_TO_VERSION_MAP from './hash_to_version_map.json';
 export interface KibanaMigratorOptions {
   client: ElasticsearchClient;
   typeRegistry: ISavedObjectTypeRegistry;
   defaultIndexTypesMap: IndexTypesMap;
+  hashToVersionMap?: Record<string, string>;
```

Comment:
NIT: I see it defaults to `HASH_TO_VERSION_MAP`, but I think I'd still rather have the new parameter not optional, wdyt?

Reply:
Sounds good to me, we'll be more consistent with similar params that are being passed 👌🏼